.mockery.yaml

```diff
@@ -3,8 +3,8 @@ inpackage: True
 disable-version-string: True
 with-expecter: True
 
-filename: "{{ trimSuffix .InterfaceFile \".go\" | base | lower }}_mock.go"
-dir: "{{ .InterfaceDir }}"
+filename: "zz_generated.{{ trimSuffix .InterfaceFile \".go\" | base | lower }}_mock.go"
```

Comment:
nit: most of our generated files have `zz_generated_` prefix. Any specific reason to use the dot here instead?

Reply:
I've taken the dot format from https://github.com/kubernetes/kubernetes/blob/e39571591d2917ca642f47ebca11253a91793c4b/.generated_files#L17.

But I agree we should stick to one naming convention. I'd lean toward renaming alle our generated ones to stick to the upstream format, but can be done in a subsequent PR. WDYT?

Reply:
Agreed. 👍